ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -1327,6 +1327,30 @@ class EclCmdActivate : public EclCmdWithQueryTarget
 
 class EclCmdUnPublish : public EclCmdWithQueryTarget
 {
+    void outputResults(const IArrayOf<IConstQuerySetQueryActionResult> &results, const char *qs)
+    {
+        ForEachItemIn(i, results)
+        {
+            IConstQuerySetQueryActionResult &item = results.item(i);
+            const char *id = item.getQueryId();
+            const char *wuid = item.getWUID();
+            if (item.getSuccess())
+            {
+                if (isEmptyString(wuid))
+                    fprintf(stdout, "\nUnpublished %s/%s\n", qs, id ? id : "");
+                else
+                    fprintf(stdout, "\nDeleted %s\n", wuid);
+            }
+            else if (item.getCode()|| item.getMessage())
+            {
+                const char *msg = item.getMessage();
+                if (isEmptyString(wuid))
+                    fprintf(stderr, "Query %s Error (%d) %s\n", id ? id : "", item.getCode(), msg ? msg : "");
```

Comment:
Do we know the error was regarding the wuid delete, or could it have been related to the unpublish of the query?

Maybe again, one message.

Error unpublished roxie/myquery, and deleteing workunit W123456789: (101) error message